pulp_container/app/models.py

```diff
@@ -11,7 +11,9 @@
 from django.conf import settings
 from django.contrib.postgres import fields
 from django.shortcuts import redirect
+from django_lifecycle import hook, AFTER_CREATE, AFTER_DELETE, AFTER_UPDATE
 
+from pulpcore.cache import Cache
```

Comment:
This caused the <https://github.com/pulp/pulp_container/actions/runs/5291865234> lint job to fail with
```
ERROR: Detected bad imports from pulpcore:
./pulp_container/app/models.py:16:from pulpcore.cache import Cache
```
So we'd need to first export `Cache` from `pulpcore.plugin`?  Is there a protocol how to coordinate interdependent PRs to `pulpcore` and `pulp_container`?

Reply:
Yes, you should be importing pulpcore facilities from the plugin namespace.

In case you want to make changes in pulpcore and then benefit from it in a plugin, follow this section: https://docs.pulpproject.org/pulpcore/contributing/git.html#requiring-other-pull-requests

Reply:
...and `SyncContentCache` is derived from `Cache` and happens to be a usable substitute here; done with <https://github.com/pulp/pulp_container/pull/1301/commits/4a4ca14a36a015414f7a422f2e4c01009f6162bf> "Use derived class SyncContentCache"